### PR TITLE
Java Conversion/Deleted Unneeded Folders

### PR DESCRIPTION
**Pull request by wchen-oxy**
-I converted the main activity, list adaptor, exercise and nutrition files to java classes
-streamlined the inheritance to reduce redundancy
-added more dummy values
-deleted unneeded folders
@justinnhli @crzeller11

### CODE REVIEW

File: GetFitLA_v3/app/src/main/java/practicum/getfitla_v3/Exercise.java
Comment (crzeller11) [inline GetFitLA_v3/app/src/main/java/practicum/getfitla_v3/Exercise.java]: Ok so as I understand it, this is the hard-coded information that will later be replaced with information pulled directly from Google Sheets. When you do implement the API, these lines will be replaced with calls from a class that you should create to organize the information from the google sheet.

File: GetFitLA_v3/app/src/main/java/practicum/getfitla_v3/Exercise.java
Comment (crzeller11) [inline GetFitLA_v3/app/src/main/java/practicum/getfitla_v3/Exercise.java]: Add a comment explaining what this array list holds.

**Comment by justinnhli** [comment]
Instead of having a single folder in the repo that everything is under, why not just have those files be directly in the repo?

File: GetFitLA_v3/app/src/main/java/practicum/getfitla_v3/List_Adaptor.java
Comment (crzeller11) [inline GetFitLA_v3/app/src/main/java/practicum/getfitla_v3/List_Adaptor.java]: Eventually, you are going to need an adapter class for every activity that implements a RecyclerView of ListView type of display. This is because each adapter organizes the data specific to the activity.

File: GetFitLA_v3/app/src/main/res/layout/list_layout.xml
Comment (crzeller11) [inline GetFitLA_v3/app/src/main/res/layout/list_layout.xml]: Always use a Relative Layout in your xmls, a relative layout allows the user to rotate their screen and the layout will auto-adjust to the new orientation.

File: GetFitLA_v3/app/src/main/res/layout/maps.xml
Comment (crzeller11) [inline GetFitLA_v3/app/src/main/res/layout/maps.xml]: Again, use a Relative Layout

**Review by crzeller11 — CHANGES_REQUESTED** [review]
Added a few small comments on layout files, and added general comments on the trajectory of the code.

**Comment by justinnhli** [comment]
I opened a pull request on your fork (blackjamx5/GetFitLA#1) which removes the nested folder. Please merge that, and make sure things still work, before dealing with Chloe's requested changes.

Note that once merged, Github will collapse/hide the comments (since they are on "outdated" code).

File: GetFitLA_v3/app/src/main/java/practicum/getfitla_v3/detail.java
Comment (justinnhli) [inline GetFitLA_v3/app/src/main/java/practicum/getfitla_v3/detail.java]: Class names should be capitalized

File: GetFitLA_v3/app/src/main/res/layout/maps.xml
Comment (wchen-oxy) [reply GetFitLA_v3/app/src/main/res/layout/maps.xml]: We've decided to not allow for a rotated screen since it makes our UI look ugly. That being said you've brought up a good point about consistency which I will address right now in an updated code

**Comment by wchen-oxy** [comment]
I finished my review and it should be okay now. # @justinnhli @crzeller11

**Comment by crzeller11** [comment]
@justinnhli I will merge this if it's ok with you

**Comment by justinnhli** [comment]
LGTM. Thanks everyone

**Pull request merged**
